### DISCUSSION TITLE
1,000 separator and null values

### DISCUSSION THREAD

**Discussion by Joebjammin** [TerriaJS/terriajs#5007] [Q&A]
Hi Folks , would appreciate help with:
1) getting thousands separator to show, if possible
2) how to use a symbol in or "null" in a column to indicate no value? For example, when the value can't be computed for reliability reasons.

Thanks.

**Comment by KeyboardSounds** [comment]
Are you talking about how data is displayed in the Feature Information table on the right hand side of the map? And which version of Terria are you using, and what kind of catalog item(s)?

In version 8 (on the `next` branch), you should be able to specify a value to display to represent NULLs in the data in your catalog.json.

https://github.com/TerriaJS/terriajs/blob/0fcc247607c2196bea4c5d3dff1b4cc03a744e64/lib/Traits/FeatureInfoTraits.ts#L93-L99

Something like:
```
{
  "name": "my catalog item",
  "type": "csv",
  "showStringIfPropertyValueIsNull": "NULL"
  // .. the rest of the fields for the catalog item
}
```

**Reply by Joebjammin** [reply]
Thank you Emma for your response to the null values question. I am most interested in thousands separator here at present:
<img width="1432" alt="2020-11-23_00-17-58" src="https://user-images.githubusercontent.com/42526991/99931214-b17f1b00-2d21-11eb-9035-29c4be50727a.png">

**Reply by zoran995** [reply]
@Joebjammin From what I know, number formatting (1000 separator) is not supported atm, there is #4191 for it.

**Reply by Joebjammin** [reply]
@zoran995 Thank you. Much appreciated.
